### PR TITLE
Delete meal foods

### PR DESCRIPTION
**Pull request by izaxveta**
#### Pivotal URL:
https://www.pivotaltracker.com/story/show/154940758

#### What does this PR do?
This PR adds the functionality to delete a mealFood record, removing the association that a food may have with a specific meal. This PR also tests edge case in which a user attempts to delete a food that has no association with a meal from that same meal. 

#### Where should the reviewer start?
Start at the foods controller with the destroy function, then continue to review further on into the model with the find function that checks if the mealFood association exists and then the delete function which actually deletes the mealFood record if the association is actually there.

#### Any additional context you want to provide?
No

#### Do migrations need to be run?
No

### CODE REVIEW

File: models/mealFood.js [hunk 1]
```diff
@@ -15,6 +23,20 @@ const MealFood = {
       })
     })
   },
+
+  delete: (mealId, foodId) => {
+    return database.raw(
+      'DELETE FROM meal_foods WHERE meal_foods.meal_id = ? AND meal_foods.food_id = ? RETURNING *',
+      [mealId, foodId]
+    ).then((response) => {
+      // var pry = require('pryjs'); eval(pry.it)
```
Comment (marshallhouston) [inline models/mealFood.js]: Commented code here.

File: controllers/mealFoodsController.js [hunk 2]
```diff
@@ -16,6 +16,27 @@ const create = (req, res, next) => {
...
+
+  Promise.all([mealFood])
+    .then(mealFood => {
+      if (!mealFood[0]) {
+        return res.sendStatus(404)
+      } else {
+        MealFood.delete(mealId, foodId)
+        .then(response => {
+          res.status(200).send({
+            message: `Successfully removed ${response[1].name} from ${response[0].name}`
```
Comment (marshallhouston) [inline controllers/mealFoodsController.js:33]: Nice work on making this dynamic and following the original API messaging.

**Review by marshallhouston — APPROVED** [review]
